draft-ietf-rats-architecture.md

```diff
@@ -1458,27 +1486,23 @@ The following example illustrates a hypothetical Background-Check Model solution
         |                |                |               |
         ~                ~                ~               ~
         |                |                |               |
-     time(HR_a)<---------+-------------time(HR_v)------>time(HR_r)
+     time(HR_a)<----time(HT)-----------time(HR_v)----->time(HR_r)
         |                |                |               |
      time(EG_a)          |                |               |
-        |----Evidence{time(EG_a)}-------->|               |
-        | {Handle1,time(EG_a)-time(VG_a)}|                |
+        |--E{H,time(VG_a)}--------------->|               |
+        |                |                |               |
         |                |             time(RG_v)         |
-        |<-----Attestation Result---------|               |
-        |   {time(RG_v),time(RX_v)}       |               |
-        |                |                                |
-        ~                ~                                ~
-        |                |                                |
-     time(HR_a')<--------'---------------------------->time(HR_r')
-        |                                                 |
-     time(RR_a)                                           /
-        |--Attestation Result{time(RX_v)-time(RG_v)}-->time(RA_r)
-        |    {Handle2, time(RR_a)-time(EG_a)}             |
-        ~                                                 ~
-        |                                                 |
-        |                                              time(OP_r)
-        |                                                 |
+        |<----------------R{H,time(RX_v)}-|               |
+        |                |                |               |
+     time(HR'_a)<---time(HT')----------time(HR'_v)---->time(HR'_r)
+        |                |                |               |
+     time(RR_a)          |                |               |
+        |--RR{H',R{H,time(RX_v)}}--------------------->time(RA_r)
```

Comment:
```suggestion
        |     Attestation Result
        |-------{H',R{H,time(RX_v)}}------------------>time(RA_r)
```

Reply:
```suggestion
        |---R{H',R{H,time(RX_v)}}--------------------->time(RA_r)
```

We need a legend. 'R' is short for Attestation Result.

The point of the illustration is that a Conceptual message with a stale handle is conveyed and that the Relying Party can detect that the Handle used in the Conceptual Message -- the relayed Attestation Result -- is not the current handle anymore.

Instead of adding Attestation Result text in the diagram, I simply removed the RR --> which renders the Relayed Attestation Result message a Attestation Message. This probably could be good enough, if the English text than captures that context.

Reply:
I think that you should leave the text in the diagram fully expanded until we can agree that the new HT math is correct.
Then, let's see if we can find better shortcuts/abbreviations.

Reply:
I agree there should be a legend. 
However, these examples are very complex and very terse for an architecture document. It would be better to IMO to number each message in the sequence diagram flows then explain them in more readable detail in a numbered prose section below the diagram.